Source/Extensions/NSMutableAttributedString+Helpers.swift

```diff
@@ -116,6 +116,18 @@ extension NSMutableAttributedString {
     }
 }
 
+// MARK: - Paragraph style
+
+public extension NSMutableAttributedString {
+    func paragraphTailTruncated() {
```

Comment:
I would add a documentation string here describing what this method does, just to maintain consistency with the library.